superset/app.py

```diff
@@ -241,14 +241,6 @@ def init_views(self) -> None:
             category_label=__("Manage"),
             category_icon="",
         )
-        appbuilder.add_view(
```

Comment:
Decided to hide this as it is redundant with the search query page

Reply:
This changes a public API - are we sure this won't break anything? Should it go through deprecation before being removed? Could this be done as a separate PR from the bugfix?